plugins/modules/cloudfront_distribution.py

```diff
@@ -568,7 +568,7 @@
       description:
         - The version of the http protocol to use for the distribution.
         - AWS defaults this to C(http2).
-        - Valid values are C(http1.1) and C(http2).
+        - Valid values are C(http1.1), C(http2), C(http2) and C(http2and3).
```

Comment:
```suggestion
        - Valid values are C(http1.1), C(http2), C(http2), C(http3) and C(http2and3).
```
? according to this https://github.com/ansible-collections/community.aws/pull/1753/files#diff-c6dcde753d4370b3a77da3ac2612f3028637b75a5622dc8c35f76c36cb697a15R1646-R1648